manticore/binary/__init__.py

```diff
@@ -105,31 +107,9 @@ def __init__(self, filename):
             assert self.interpreter.arch == self.arch
             assert self.interpreter.elf.header.e_type in ['ET_DYN', 'ET_EXEC']
 
-
-    def maps(self):
-        for elf_segment in self.elf.iter_segments():
-            if elf_segment.header.p_type != 'PT_LOAD' or elf_segment.header.p_memsz == 0:
-                continue
-
-            flags = elf_segment.header.p_flags
-            #PF_X 0x1 Execute - PF_W 0x2 Write - PF_R 0x4 Read
-            perms = ['   ', '  x', ' w ', ' wx', 'r  ', 'r x', 'rw ', 'rwx'][flags&7]
-            if 'r' not in perms:
-                raise Exception("Not readable map from cgc elf not supported")
-
-            #CGCMAP--
-            assert elf_segment.header.p_filesz != 0 or elf_segment.header.p_memsz != 0 
-            yield((elf_segment.header.p_vaddr,
-                  elf_segment.header.p_memsz,
-                  perms, 
-                  elf_segment.stream.name, elf_segment.header.p_offset, elf_segment.header.p_filesz))
-
     def getInterpreter(self):
         return self.interpreter
 
-    def threads(self):
```

Comment:
I only removed it because it was unused, if you feel like it's a useful API I can definitely add it back